kotlin/Tries/InsertAndSearchWordsWithWildcards.kt

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
If the data structure is uncommented in the python solution, it means that implementing the data structure was part of the solution, so we'd like it to be uncommented here as well. 

The same convention also applies for the Linked Lists chapter.

Reply:
Uncommented and noted.